inputfiles/addedTypes.jsonc

```diff
@@ -1212,7 +1212,7 @@
                 "name": "InsertPosition"
             },
             {
-                "additionalTypes": ["Headers"],
```

Comment:
`overrideType` should be a last resort especially when the original type is complex. This works:

```json
                "type": {
                    "0": {
                        "subtype": {
                            "overrideType": "[string, string]"
                        }
                    }
                },
                "additionalTypes": ["Headers"],
```

Reply:
done